server/src/TPlayer/TPlayer.cpp

```diff
@@ -2287,7 +2339,7 @@ bool TPlayer::msgPLI_LOGIN(CString& pPacket)
 	if (versionID == CLVER_UNKNOWN) {
 		// Get Iterator-Key
 		// 2.19+ RC and any client should get the key.
-		if ( isClient() || (isRC() && in_codec.getGen() > ENCRYPT_GEN_3) || getKey == true ) {
+		if ( (isClient() && type != PLTYPE_WEB) || (isRC() && in_codec.getGen() > ENCRYPT_GEN_3) || getKey == true ) {
```

Comment:
is it right to say this shouldn't happen for anything < encrypt_gen_2 rather than use the type != web

Reply:
actually looking at the code, scratch that - as it is, is fine